plugins/modules/cloud/scaleway/scaleway_private_network.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway VPC management module
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_private_network
+short_description: Scaleway private network management module
+version_added: 4.4.0
```

Comment:
The next release will be 4.5.0:
```suggestion
version_added: 4.5.0
```